project_event/models/project_task.py

```diff
@@ -378,14 +378,26 @@ def get_message_body(self, action):
         return switcher.get(action)
 
     def get_message(self, action):
+        message = '<br>'
+        responsible = self.responsible_id.id
+        if self.activity_task_type == 'activity':
+            message += _('Avtivity: <br>') + self.name + '<br>'
```

Comment:
message += _('Avtivity: <br>') + self.name + '<br>' n check the spelling

Reply:
Done, thanks